### PR TITLE
Add Chronos ansible role

### PR DESCRIPTION
**Pull request by tymofii-polekhin**
Chronos
=======

[Chronos](http://mesos.github.io/chronos) It is a distributed and fault-tolerant scheduler that runs on top of Apache Mesos that can be used for job orchestration. It supports custom Mesos executors as well as the default command executor. 
Thus by default, Chronos executes sh (on most systems bash) scripts.

Chronos listens on port 4400.

### CODE REVIEW

File: roles/chronos/README.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Chronos
+=======
+
+.. versionadded:: 0.1
+
+`Chronos <http://http://mesos.github.io/chronos/>`_ It is a distributed 
+and fault-tolerant scheduler that runs on top of Apache Mesos that can 
+be used for job orchestration. It supports custom Mesos executors as well 
+as the default command executor. 
+Thus by default, Chronos executes sh (on most systems bash) scripts.
```
Comment (BrianHicks) [inline roles/chronos/README.rst]: should be "shell scripts" instead of "sh"

File: roles/chronos/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
+# right now there is an issue with chronos zookeeper connection with auth, so leave it blank for now
```
Comment (BrianHicks) [inline roles/chronos/defaults/main.yml]: can you be more specific here? What's the issue? If there's an open ticket, could you link to it please?

**Comment by BrianHicks** [comment]
- [ ] needs documentation in the form of...
  1. documentation for all the role variables
  2. a stub in `docs/components`
  3. a link in the toctree of `docs/components/index.rst`
- [ ] chronos should be added to the control role of `terraform.sample.yml` as well as the comments there, where appropriate.
- [ ] needs documentation for the new options for `security-setup`.

Looks pretty good otherwise. I'll test once you add these things :+1:

File: roles/chronos/defaults/main.yml (hunk 2 above)
Comment (tymofii-polekhin) [inline roles/chronos/defaults/main.yml]: If we run a chronos with `--zk-hosts chronos:8WDxe30zbCzcuZJc@zookeeper.service.consul:2181` param we get the following error:
```
May 29 09:52:09 clean-deploy-test-01 chronos[17850]: 2015-05-29 09:52:09,300:17850(0x7fd7f1374700):ZOO_INFO@zookeeper_init@786: Initiating client connection, host=chronos:8WDxe30zbCzcuZJc@z
ookeeper.service.consul:2181 sessionTimeout=10000 watcher=0x7fd7fae3c1e0 sessionId=0 sessionPasswd=<null> context=0x7fd818021d40 flags=0
May 29 09:52:09 clean-deploy-test-01 chronos[17850]: F0529 09:52:09.333401 18132 zookeeper.cpp:113] Failed to create ZooKeeper, zookeeper_init: No such file or directory [2]
```

`host=chronos:8WDxe30zbCzcuZJc@zookeeper.service.consul:2181` -  i think this is the cause of the issue.

The marathon log looks a little different:
```
May 29 10:12:27 clean-deploy-test-01 marathon[20835]: 2015-05-29 10:12:27,840:20835(0x7fb639368700):ZOO_INFO@zookeeper_init@786: Initiating client connection, host=zookeeper.service.consul:
2181 sessionTimeout=10000 watcher=0x7fb642e3c1e0 sessionId=0 sessionPasswd=<null> context=0x7fb648beeb00 flags=0
```

i cant say if marathon just drops the auth part and this makes it start, or its an error in the chronos args parser

**Comment by tymofii-polekhin** [comment]
@BrianHicks are you talking about variables is security.yml?
new option in security-setup have help in them
there is no terraform in this branch yet

**Comment by BrianHicks** [comment]
No, the variables in `defaults/main.yml` in the new role.

We document the options both places. See `docs/security/security_setup.rst`.

I'll open an issue for the Terraform additions since we don't have them in this branch, yet.

**Comment by OleksandrBerchenko** [comment]
It's finally reviewed, tested and merged into qa/features-master-integration branch.

If there are no other issues, it can probably be merged into master branch (+ update terraform.sample.yml).

**Comment by OleksandrBerchenko** [comment]
@BrianHicks, what are our next steps?  Should we open a PR to master?

Thanks!

**Comment by tymofii-polekhin** [comment]
PR to master is already open, and Brian will merge it into 0.3.1 version

**Pull request merged**